### PR TITLE
added a new chart for crd resources

### LINKED ISSUE #2109: Helm chart for cluster provisioning
**Is this a bug report or feature request?**
* Feature Request

**What should the feature do:**
I'd like to have helm chart for cluster creating. Both for sc and cluster itself
https://rook.io/docs/rook/master/block.html
https://github.com/rook/rook/blob/master/Documentation/ceph-quickstart.md#create-a-rook-cluster

**What is use case behind this feature:**
I can have all the parts of the env managed by helm

### PR DESCRIPTION
**Pull request by saravanakumar-periyasamy**
Signed-off-by: Saravana <saravana.periyasamy@target.com>

Description of your changes:

Which issue is resolved by this Pull Request:
Resolves #2109 

Checklist:
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.

### CODE REVIEW

File: cluster/charts/rook-resources/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: rook-resources
```
Comment (travisn) [inline cluster/charts/rook-resources/Chart.yaml]: what if the chart name matched the top-level crd name? For example: `rook-cluster`

Comment (saravanakumar-periyasamy) [reply cluster/charts/rook-resources/Chart.yaml]: ya, `rook-cluster` sounds better. I will update this.

**Comment by saravanakumar-periyasamy** [comment]
sorry I could not get to it.. I will add the docs & update the PR

**Comment by sebastien-prudhomme** [comment]
Hi @saravanakumar-periyasamy, i'm testing your PR.

As the API namespace has changed, you need to correct the code: rook.io is now ceph.rook.io

**Comment by sebastien-prudhomme** [comment]
Another point: the toolbox image name and image tag should be configurable in values.yaml

**Comment by sebastien-prudhomme** [comment]
@saravanakumar-periyasamy I've create a PR in your fork so that it works fine with Rook master branch. A lot has changed in the RBAC model.

**Comment by jbw976** [comment]
This PR looks to address the newly opened issue for Ceph cluster helm chart #2109. Now that the Ceph CRDs are in Beta and are more stable, we should revive this helm chart effort for them.  This would be appreciated by the community if we have more time to work on it again @saravanakumar-periyasamy!

**Comment by saravanakumar-periyasamy** [comment]
@jbw976 sorry, my priorities got changed and could not get back to this.. I will try to spend sometime over the weekend and work on this again.

**Comment by florianrusch** [comment]
@saravanakumar-periyasamy if you need help, let me know ;-) I'm also looking forward to use this chart :-)

**Comment by travisn** [comment]
@saravanakumar-periyasamy @florianrusch It would still be awesome to have a cluster crd chart! Can we revive this one? There are a number of changes that would be needed since the last couple of releases. Ping me when you're ready to talk.

**Comment by travisn** [comment]
With the yaml redesign in #2820, instead of creating a new helm chart I'm thinking we can just combine this work with the existing helm chart. The default/mainline case will be to have the operator and cluster in the same namespace, so this seems like a natural fit.

**Comment by travisn** [comment]
Closing this PR since it's getting so old. With the merge of #2820 we also want a new, simplified approach to the helm chart. The changes in this PR will get us most of the way there and will be a great reference. Comments are added to #2109 with the new approach.

**Pull request closed without merging**